src/Cli/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx

```diff
@@ -232,4 +232,10 @@ If you would like to create a manifest, use `dotnet new tool-manifest`, usually
   <data name="PrereleaseAndVersionAreNotSupportedAtTheSameTime" xml:space="preserve">
     <value>The --prerelease and --version options are not supported in the same command</value>
   </data>
+  <data name="CreateManifestIfNeededOptionDescription" xml:space="preserve">
+    <value>Create tool manifest when installing tools if manifest not exist.</value>
+  </data>
+  <data name="CreateManifestIfNeededOptionName" xml:space="preserve">
+    <value>MANIFEST</value>
+  </data>
```

Comment:
Looks like we don't need this string and can delete it (see comment in ToolInstallCommandParser.cs).

```suggestion
```